pysamstats/test/test_io.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+import logging
+
+from pysam import Samfile
+from pysamstats import load_coverage, stat_coverage
+
+from pysamstats.io import write_hdf5
+import tables
+
+logger = logging.getLogger(__name__)
+debug = logger.debug
+
+
+def write_hdf5_dtype(arg):
```

Comment:
Not sure it's worth breaking this out into a separate function, it's only ever called from one place. Don't mind though. If it is separate, I'd be inclined to call it something like "_test_write_hdf5_chrom_dtype" or "check_write_hdf5_chrom_dtype" to be descriptive.